ocaml/testsuite/tests/typing-layouts-bits32/basics_alpha.ml

```diff
@@ -0,0 +1,689 @@
+(* TEST
+   flags = "-extension layouts_alpha"
+   * expect
+*)
+
+(* This file contains typing tests for the layout [bits32].
```

Comment:
I do think having tests for every layout is adding _something_: We're going to be moving different layouts through different extension flags at different paces, and there are some core differences among the layouts (e.g., because of all-float records).  It was very useful to have the `float64` tests as I've been working on moving float to `layouts_beta`, because the type system now checks the extension level in many different places (sadly) and I would not have found them all otherwise.

To alleviate the maintenance burden, we can just decide to not hold ourselves to a super high standard in terms of cleanliness and review, here.  I think the main value of these tests is as a sanity check for future changes.

One place where we can save on duplication is in `typing_layouts/basics_*.ml`.  This pr doesn't update those tests, and I think that's fine.  I am going to update them in the PR that moves `float64` to layouts beta, so that they include tests for at least one non-void non-value sort (there are CRs in there suggesting to do this), but I don't think we need to update them for _every_ new sort.

Now, that file does overlap in purpose somewhat with these layout-specific `basics` files.  But not entirely, and I think it's probably just not worth the effort to unify and rationalize them.